pkg/virt-launcher/virtwrap/network/dhcp/dhcp.go

```diff
@@ -174,12 +182,12 @@ func (h *DHCPHandler) ServeDHCP(p dhcp.Packet, msgType dhcp.MessageType, options
 	case dhcp.Discover:
 		log.Log.V(4).Info("The request has message type DISCOVER")
 		return dhcp.ReplyPacket(p, dhcp.Offer, h.serverIP, h.clientIP, h.leaseDuration,
-			h.options.SelectOrderOrAll(options[dhcp.OptionParameterRequestList]))
+			h.options.SelectOrderOrAll(nil))
```

Comment:
I'm unsure about what change in behavior this might cause. @vladikr does ordering matter here?

Reply:
@bharatnc why you have to change this?

Reply:
@bharatnc looking at the source,
https://github.com/krolaw/dhcp4/blob/master/helpers.go#L9-L33

The implementation suggests that the `order` only the options present in the order presented in the byte array.  Which options get stripped and why do you expect them to be present (if not present in `options[dhcp.OptionParameterRequestList]` (the `order` byte array)

Reply:
The extra dhcp options that we provide from [here](https://github.com/kubevirt/kubevirt/pull/1848/files#diff-4cc299b396934683b5f4388deaff9e77R813), gets stripped if we do not change this. This is needed to configure options that are reserved for private use 224-254.

Reply:
Note: we’re mitigating the Select part of this - the Order is less important.

Reply:
@bharatnc as far as I know they need to be stripped to be standard compliant if they were not requested. Can you make sure that hour dhcp agent requests the options instead? That should be the expected behaviour.

Reply:
@rmohr , thanks for your review. By ``our dhcp agent`` I assume that you want me to implement this change in another part of the repo. I did not quite get where you want this change instead. Could point me to there? I appreciate it!

Reply:
@rmohr, apologies for the delay in this reply, I rebased this PR today. I realized that this behavior using ``h.options.SelectOrderOrAll(nil))`` has been implemented upstream already, so this PR isn't introducing this change now - after the rebase.